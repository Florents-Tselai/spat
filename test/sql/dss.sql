SELECT dss_echo('hello world');
