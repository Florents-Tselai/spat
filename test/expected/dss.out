SELECT dss_echo('hello world');
  dss_echo   
-------------
 hello world
(1 row)

